oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollector.java

```diff
@@ -31,6 +32,34 @@ public interface FullGCStatsCollector {
      */
     void documentRead();
 
+    /**
+     * Total No. of properties deleted during a given GC phase
+     * @param mode GC phase
+     * @param numProps no. of properties deleted in current cycle
+     */
+    void collectedPropertiesDeleted(GCPhase mode, long numProps);
```

Comment:
this method is a duplicate of `void propertiesDeleted(long numProps);`. I don't get the rationale behind this.

Reply:
I would rename this to `propertiesCollected` or `collectedProperties`

Reply:
Yes, makes more sense